hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/utils/FederationStateStoreFacade.java

```diff
@@ -968,7 +969,7 @@ public int getActiveSubClustersCount() throws YarnException {
    * @throws YarnException When there is no Active SubCluster,
    * an exception will be thrown (No active SubCluster available to submit the request.)
    */
-  public static SubClusterId getRandomActiveSubCluster(
+  public SubClusterId getRandomActiveSubCluster(
```

Comment:
Isn't this still static?

Reply:
We should change the places where we call it to call it as static.

Reply:
Thank you very much for your help to review the code! This part of the code can continue to be static, I will modify the code.